esp/useful_scripts/server_setup/new_site.sh

```diff
@@ -397,19 +388,12 @@ WSGIDaemonProcess $SITENAME processes=1 threads=1 maximum-requests=1000
     #   Caching - should use Squid if performance is really important
     # CacheEnable disk /
 
+    #   Redirect HTTP requests to HTTPS for security - uncomment to use
+    # Include /etc/apache2/sites-available/esp_sites/https_redirect.conf
```

Comment:
I had it commented out since I was afraid to make assumptions about the server configuration (e.g. that https_redirect.conf exists in the right place), but it looks like we make plenty of those assumptions elsewhere.  I think the right solution is to just put the code inline, since it's only 4 lines.